src/commands/command.ts

```diff
@@ -6,6 +7,7 @@ export type TriggerCriteria = string | RegExp | ((message: Message, keyword: str
 export interface ConcreteTrigger {
     command: Command,
     criteria: TriggerCriteria,
+    args: string[],
```

Comment:
I thought we said we weren't adding this?

Reply:
Right - my bad. I changed that before we discussed that problem in detail to get it working, since - as we had it before it would slice off the prefix length regardless of whether it's a `string`-trigger or not. I'll change that in a second.  
I'll just drag the line with `trigger.args = message.content.split(/ +/g);` back into `message.handler` and adapt all methods accordingly.